app/uk/gov/hmrc/economiccrimelevyregistration/views/CheckYourAnswersView.scala.html

```diff
@@ -15,15 +15,29 @@
  *@
 
 @this(
-    layout: templates.Layout,
-    govukSummaryList: GovukSummaryList
+        layout: templates.Layout,
+        govukSummaryList: GovukSummaryList,
+        govukButton: GovukButton
 )
 
-@(list: SummaryList)(implicit request: Request[_], messages: Messages)
+@(organisationDetails: SummaryList, personalDetails: SummaryList)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(messages("checkYourAnswers.title"))) {
+@layout(pageTitle = title(messages("checkYourAnswers.title")), backLinkUrl = Some(StartController.onPageLoad().url)) {
```

Comment:
Is it correct that we go back to the start if the back link is clicked?

Reply:
No - good spot, I've fixed this in the latest push :)